2019-11-15 bioinformatics and python meetup at python

maria and husband on bioinformatics, and chatbots and noprofits

FPKM data is already normalized
Fragment per kilo per millions
need to normalize for the amplification variable
UMI unique molecule identification, add artifical molecules to MRNAs to identify particular RNAs and then see how much they are amplified, you can count the originals before

without UMI you don't know if they are 2 MRNAs from the same place or

R or DC2 or equivalent which takes into account biology of genes based on variability in each gene, high and low variability
HTseq for counting count matrix genese
star for alignment gives you a BAM
enrichr compares gene to lots of databases of chemical pathways etc.
e-seq annotations and counts


